pyEpiabm/pyEpiabm/routine/file_population_config.py

```diff
@@ -175,18 +182,18 @@ def add_households(microcell: Microcell, household_number: int):
         """
         # Initialises another multinomial distribution
```

Comment:
Is this a duplicate of the method in the toy_pop config? Maybe we can import that in here rather than having duplicated code

Reply:
Its not quite a duplicate. In the file population the method takes a microcell whereas in toy population it takes a population. I could call the method from the file population into toy population at the level of the microcell. Also the way q is written it is always 1 so I wasn't sure if it was worth removing that line altogether?